tests/integration/ts_runtime_test.go

```
@@ -36,7 +36,7 @@ func TestTypeScriptRuntimeIntegration(t *testing.T) {
 
 	// Step 1: Type-check with tsc --noEmit
 	t.Run("tsc_typecheck", func(t *testing.T) {
-		cmd := exec.Command("npx", "tsc", "--noEmit")
+		cmd := exec.Command("npx", "tsc", "--noEmit", "--project", "tsconfig.json")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)